go-controller/pkg/network-controller-manager/network_controller_manager.go

```diff
@@ -242,10 +244,49 @@ func (cm *networkControllerManager) configureMetrics(stopChan <-chan struct{}) {
 	metrics.MonitorIPSec(cm.nbClient)
 }
 
+func (cm *networkControllerManager) createACLLoggingMeter() error {
+	band := &nbdb.MeterBand{
+		Action: ovntypes.MeterAction,
+		Rate:   config.Logging.ACLLoggingRateLimit,
+	}
+	ops, err := libovsdbops.CreateMeterBandOps(cm.nbClient, nil, band)
+	if err != nil {
+		return fmt.Errorf("can't create meter band %v: %v", band, err)
+	}
+
+	meterFairness := true
+	meter := &nbdb.Meter{
+		Name: ovntypes.OvnACLLoggingMeter,
+		Fair: &meterFairness,
+		Unit: ovntypes.PacketsPerSecond,
+	}
+	ops, err = libovsdbops.CreateOrUpdateMeterOps(cm.nbClient, ops, meter, []*nbdb.MeterBand{band},
+		&meter.Bands, &meter.Fair, &meter.Unit)
+	if err != nil {
+		return fmt.Errorf("can't create meter %v: %v", meter, err)
+	}
+
+	_, err = libovsdbops.TransactAndCheck(cm.nbClient, ops)
+	if err != nil {
+		return fmt.Errorf("can't transact ACL logging meter: %v", err)
+	}
+
+	return nil
+}
+
+func (cm *networkControllerManager) configureAclLoggingSupport() {
```

Comment:
Yeah, why not handle the error as we do for example `enableOVNLogicalDataPathGroups`?
Multicast is different because what we are doing there is checking if OVN supports it, but here it is just a random error?

Reply:
@jcaamano @maiqueb  I am keeping the current behavior for default network. see https://github.com/ovn-org/ovn-kubernetes/blob/43da7b1a7c62ba78fffa14965a4bee552d24dc11/go-controller/pkg/ovn/default_network_controller.go#L326